onnxruntime/core/platform/windows/env.cc

```diff
@@ -217,7 +217,7 @@ class WindowsThread : public EnvThread {
   }
 #pragma warning(pop)
 
-  static void __stdcall CustomThreadMain(void* param) {
+  static void CustomThreadMain(void* param) {
```

Comment:
_beginthreadex expects a __stdcall function pointer. Do we still use _beginthreadex  to create threads?

Reply:
This function is passed to custom thread creation function which requires `__cdecl`. Our making this function `__stdcall` and then casting causes undefined behavior, crash in a unittest in 32-bit builds and consequently a build failure. We are shipping 32-bit with stack corruption (and possibly 64-bit, but there the args are passed through registers).